template_snippets/cloudconfig/g8s_master.yaml

```diff
@@ -39,8 +39,8 @@
     content: |
       [Unit]
       Description=g8s-master-api-certs
-      Requires=docker.service
-      After=docker.service
+      Wants=calico-node.service
```

Comment:
I wanted to have all the chain depending of calico for consistency. Ofc I don't need calico to run the certificates but I want to create a chain with these units being calico-node the initial one.